testsuite/tests/templates/basic/test.ml

```diff
@@ -32,11 +40,14 @@
    ref_indirect.ml \
    ref_indirect.cmo.ocamlobjinfo.reference \
    ref_indirect.cmx.ocamlobjinfo.reference \
+   run.ml \
```

Comment:
is this file intentionally empty?

Reply:
Oh heh no, I must've had some idea for what it was going to be and changed my mind. Deleting